.github/workflows/check-catalyst.yaml

```diff
@@ -75,6 +75,8 @@ jobs:
           QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
           QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
           ENABLE_LIGHTNING_KOKKOS=ON \
+          CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" \
+          ENABLE_OPENQASM=ON \
```

Comment:
I also included the OpenQASM backend here in the runtime build job, rather than further down in the OQ3 test job, since that's how we do it for kokkos as well. Or was there a specific reason to rebuild the runtime in the test job? :)

Reply:
The build with OQ3 is not sufficient as of its dependency to the Amazon Braket Python SDK. I didn't want to end up building something that's partially working in this GH job, so ended up building the runtime in the OQ3 test job. Also, building runtime is quite fast with or without this backend, so there wasn't any remarkable changes to the CI total time.